src/lib/mina.ts

```diff
@@ -677,22 +677,34 @@ LocalBlockchain satisfies (...args: any) => Mina;
  * Represents the Mina blockchain running on a real network
  */
 function Network(graphqlEndpoint: string): Mina;
-function Network(graphqlEndpoints: {
+function Network(endpoints: {
+  mina: string | string[];
+  accountsManager?: string;
+}): Mina;
```

Comment:
I don't think it makes sense to have a separate function signature which includes `accountsManager` but excludes `archive` -- please delete. Instead, make `archive` optional in the one below